nomos-services/network/src/backends/libp2p/command.rs

```diff
@@ -21,6 +21,29 @@ pub enum Command {
         message: Box<[u8]>,
         retry_count: usize,
     },
+
+    // Kademlia commands
+    AddPeer {
+        peer_id: PeerId,
+        addr: Multiaddr,
+        reply: oneshot::Sender<Result<(), String>>,
+    },
```

Comment:
Nice, we would have to call this when loading federated trusted nodes from config probably. So we can bootstrap the network.